tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     "declaration": true,
-    "target": "es2017",
+    "target": "es2018",
```

Comment:
This is essentially a breaking change, any reason this was added?

Reply:
I guess because the build suggested this as a fix for using `finally` on promises. It's not necessarily a bad thing, but it does mean that anyone consuming the library will have to be aware that this was the target for the es modules.

Reply:
@victoringvarsson you can add the `es2018` again, and mention in the commit that this is a breaking change.

Reply:
Fixed.